default-plugins/tab-bar/src/tab.rs

```diff
@@ -40,12 +40,19 @@ pub fn non_active_tab(text: String, palette: Palette, separator: &str) -> LinePa
 pub fn tab_style(
     text: String,
     is_active_tab: bool,
+    is_fullscreen_active: bool,
     is_sync_panes_active: bool,
+    panes: usize,
     palette: Palette,
     capabilities: PluginCapabilities,
 ) -> LinePart {
     let separator = tab_separator(capabilities);
     let mut tab_text = text;
+    if is_fullscreen_active {
+        let current_panes = panes - 2;
+        let fullscreen_text = format!(" (Fullscreen +{} hidden panes)", current_panes);
```

Comment:
Do you think it will be possible to have this in the status-bar instead? Maybe instead of the "Tip" line when we're in full-screen? I feel it's a better place for it.

Reply:
I'd second this! That was my first thought when seeing this :)

Reply:
I think so too, so I'll try to fix that!